src/converters/app-converter.ts

```diff
@@ -367,10 +448,39 @@ export class StackConverter extends ArtifactConverter implements intrinsics.Intr
     /** @internal */
     asOutputValue<T>(v: T): T {
```

Comment:
Cutting a follow up ticket for handling cases where multiple nested stacks have a resource with the same logical ID. The users could inform the converter about the right stack by passing it in as an optional argument.